src/views/users/Role/RoleList/BaseRoleList.vue

```diff
@@ -70,7 +70,7 @@ export default {
                   name: 'RoleDetail',
                   query: {
                     tab: 'RoleUsers',
-                    scope: row.scope.value
+                    _scope: row.scope.value
                   },
                   params: {
                     id: row.id
```

Comment:
The above template and script appear to be two separate pieces of code without any direct comparison. To provide an efficient answer, I would need information about which specific elements or portions of these templates and scripts are being compared against each other. Please let me know the sections you intend to compare or if there's something else more specific that needs analysis.